tests/commands/run_test.py

```diff
@@ -59,6 +59,7 @@ def _get_opts(
         origin='',
         source='',
         allow_unstaged_config=False,
+        hook_stage=[]
```

Comment:
probably not a bug in this case, but default values that are mutable can cause issues:

http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments